<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>278</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="1">
         <layout class="QGridLayout" name="gridLayout_10">
          <item row="11" column="0">
           <widget class="QPushButton" name="finishButton">
            <property name="text">
             <string>Завершить тестирование</string>
            </property>
            <property name="autoExclusive">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Крен</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="krenGradSpinBox">
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
              <property name="buttonSymbols">
               <enum>QAbstractSpinBox::NoButtons</enum>
              </property>
              <property name="suffix">
               <string>°</string>
              </property>
              <property name="minimum">
               <number>-90</number>
              </property>
              <property name="maximum">
               <number>90</number>
              </property>
              <property name="value">
               <number>-90</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="krenMinSpinBox">
              <property name="readOnly">
               <bool>true</bool>
              </property>
              <property name="buttonSymbols">
               <enum>QAbstractSpinBox::NoButtons</enum>
              </property>
              <property name="suffix">
               <string>'</string>
              </property>
              <property name="maximum">
               <number>59</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="krenCheckBox">
              <property name="text">
               <string>Добавить</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="5" column="0">
           <widget class="QPushButton" name="saveInfoButt">
            <property name="text">
             <string>Остановить и сохранить текущие значения</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QPushButton" name="loadInfoButt">
            <property name="text">
             <string>Загрузить последние сохраненные данные</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Долгота</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="dolgGradSpinBox">
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
              <property name="buttonSymbols">
               <enum>QAbstractSpinBox::NoButtons</enum>
              </property>
              <property name="suffix">
               <string>°</string>
              </property>
              <property name="minimum">
               <number>-180</number>
              </property>
              <property name="maximum">
               <number>180</number>
              </property>
              <property name="value">
               <number>-180</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="dolgMinSpinBox">
              <property name="readOnly">
               <bool>true</bool>
              </property>
              <property name="buttonSymbols">
               <enum>QAbstractSpinBox::NoButtons</enum>
              </property>
              <property name="suffix">
               <string>'</string>
              </property>
              <property name="maximum">
               <number>59</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="dolgotaCheckBox">
              <property name="text">
               <string>Добавить</string>
              </property>
              <property name="checked">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="12" column="0">
           <widget class="QPushButton" name="openErrorListBtn">
            <property name="text">
             <string>Открыть журнал об ошибках</string>
            </property>
            <property name="checkable">
             <bool>false</bool>
            </property>
            <property name="autoExclusive">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Дифферент</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="diffGradSpinBox">
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
              <property name="buttonSymbols">
               <enum>QAbstractSpinBox::NoButtons</enum>
              </property>
              <property name="suffix">
               <string>°</string>
              </property>
              <property name="minimum">
               <number>-90</number>
              </property>
              <property name="maximum">
               <number>90</number>
              </property>
              <property name="value">
               <number>-90</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="diffMinSpinBox">
              <property name="readOnly">
               <bool>true</bool>
              </property>
              <property name="buttonSymbols">
               <enum>QAbstractSpinBox::NoButtons</enum>
              </property>
              <property name="suffix">
               <string>'</string>
              </property>
              <property name="maximum">
               <number>59</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="differentCheckBox">
              <property name="text">
               <string>Добавить</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="8" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_8">
              <item>
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>Время тестирования</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_7">
                <item>
                 <widget class="QLabel" name="label_7">
                  <property name="text">
                   <string>Часы</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="hoursBox">
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                  <property name="buttonSymbols">
                   <enum>QAbstractSpinBox::NoButtons</enum>
                  </property>
                  <property name="maximum">
                   <number>100</number>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_8">
                  <property name="text">
                   <string>Минуты</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="minutesBox">
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                  <property name="buttonSymbols">
                   <enum>QAbstractSpinBox::NoButtons</enum>
                  </property>
                  <property name="maximum">
                   <number>59</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item row="2" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Курс</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="kursGradSpinBox">
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
              <property name="buttonSymbols">
               <enum>QAbstractSpinBox::NoButtons</enum>
              </property>
              <property name="suffix">
               <string>°</string>
              </property>
              <property name="minimum">
               <number>0</number>
              </property>
              <property name="maximum">
               <number>360</number>
              </property>
              <property name="value">
               <number>0</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="kursMinSpinBox">
              <property name="readOnly">
               <bool>true</bool>
              </property>
              <property name="buttonSymbols">
               <enum>QAbstractSpinBox::NoButtons</enum>
              </property>
              <property name="suffix">
               <string>'</string>
              </property>
              <property name="maximum">
               <number>59</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="kursCheckBox">
              <property name="text">
               <string>Добавить</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="7" column="0">
           <widget class="QPushButton" name="errorButton">
            <property name="text">
             <string>Отказ</string>
            </property>
           </widget>
          </item>
          <item row="10" column="0">
           <widget class="QPushButton" name="continueButton">
            <property name="text">
             <string>Продолжить тестирование</string>
            </property>
            <property name="autoExclusive">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Широта</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="shirGradSpinBox">
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
              <property name="buttonSymbols">
               <enum>QAbstractSpinBox::NoButtons</enum>
              </property>
              <property name="suffix">
               <string>°</string>
              </property>
              <property name="prefix">
               <string/>
              </property>
              <property name="minimum">
               <number>-90</number>
              </property>
              <property name="maximum">
               <number>90</number>
              </property>
              <property name="value">
               <number>-90</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="shirMinSpinBox">
              <property name="readOnly">
               <bool>true</bool>
              </property>
              <property name="buttonSymbols">
               <enum>QAbstractSpinBox::NoButtons</enum>
              </property>
              <property name="suffix">
               <string>'</string>
              </property>
              <property name="maximum">
               <number>59</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="shirotaCheckBox">
              <property name="text">
               <string>Добавить </string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="9" column="0">
           <widget class="QPushButton" name="startBut">
            <property name="text">
             <string>Начать тестирование</string>
            </property>
            <property name="autoExclusive">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="13" column="0">
           <widget class="QProgressBar" name="progressBar">
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="value">
             <number>0</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QTextEdit" name="displayTextEdit">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="backButton">
            <property name="text">
             <string>Назад</string>
            </property>
            <property name="autoExclusive">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>278</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
